src/main/java/org/jenkinsci/backend/ircbot/IrcListener.java

```diff
@@ -414,19 +429,27 @@ private void setupHosting(Channel channel, User sender, String hostingId, String
                 return;
             }
 
+            String prUrl = createUploadPermissionPR(channel,sender,issueID,forkTo,users,Collections.singletonList(defaultAssignee));
+            if(StringUtils.isBlank(prUrl)) {
+                out.message("Could not create upload permission pull request");
+            }
+
             // update the issue with information on next steps
             String msg = "At the request of " + sender.getNick() + " on IRC, "
                     + "the code has been forked into the jenkinsci project on GitHub as "
                     + "https://github.com/jenkinsci/" + forkTo
                     + "\n\nA JIRA component named " + forkTo + " has also been created with "
                     + defaultAssignee + " as the default assignee for issues."
-                    + "\n\nPlease remove your original repository so that the jenkinsci repository "
-                    + "is the definitive source for the code. Also, please make sure you properly "
-                    + "follow the [documentation on documenting your plugin|https://jenkins.io/doc/developer/publishing/documentation/] "
+                    + "\n\nA [pull request|" + prUrl +"] has been created against the repository permissions updater to "
```

Comment:
may be overkill but you're handling the PR url not being created but this text here is misleading, it will just link to an empty url

Reply:
Good catch, I'll fix that up